UPGRADE-1.0.md

```diff
@@ -0,0 +1,129 @@
+# Upgrade to dd-trace-php 1.0
+
+**Important Notice:** Upgrading `dd-trace-php` from version 0.x to 1.0 involves several breaking changes. Please review the following changes carefully before proceeding with the upgrade.
+
+
+Version 1.0 of dd-trace-php brings significant enhancements and improvements over the previous versions. Alongside various bug fixes and performance optimizations, this release introduces a revamped WordPress integration, modifications to defaults, and deprecation of OpenTracing in favor of OpenTelemetry.
+
+---
+
+## Changes Summary
+
+- [PHP 5 EOL](#php-5-eol)
+- [WordPress](#wordpress)
+    - [Configuration Changes](#configuration-changes)
+    - [Breaking Changes](#breaking-changes)
+- [Context Propagation](#context-propagation)
+- [Sampling Rules](#sampling-rules)
+- [OpenTracing Deprecation](#opentracing-deprecation)
+- [Removed APIs and Configuration Keys](#removed-apis-and-configuration-keys)
+
+## Migration Guide
+
+To ensure a smooth transition to version 1.0, follow these migration steps:
+- **Check PHP Version**: Ensure your PHP version is at least 7, as PHP 5 is no longer supported starting from version 1.0.
+- **Update WordPress Integration**: If you're using the WordPress integration, adjust your configuration and monitors based on the changes outlined in the document.
+- **Review Sampling Rules**: Ensure your sampling rules are configured correctly, considering the switch from regex patterns to glob patterns.
+- **Update Deprecated APIs and Configuration Keys**: Replace deprecated functions, interfaces, classes, and configuration keys with their respective replacements as detailed in the document.
+- **Test Thoroughly**: Test your application thoroughly after the upgrade to ensure that all functionalities are working as expected.
+
+---
+
+## PHP 5 EOL
+
+Starting from version 1.0, support for PHP 5 is removed. If you are using PHP 5, you can still use the PHP tracer up to version 0.99. The minimum PHP version requirement for `dd-trace-php` 1.x is PHP 7.
+
+## WordPress
+
+An opt-in improvement to the WordPress integration was released as a public-beta in [0.91.0](https://github.com/DataDog/dd-trace-php/releases/tag/0.91.0). Starting version 1.0, the PHP tracer will adopt this integration.
+
+### Configuration Changes
+
+| Configuration Option                      | Previous Configuration Value | New Configuration Value |
+|-------------------------------------------|------------------------------|-------------------------|
+| `DD_TRACE_WORDPRESS_ENHANCED_INTEGRATION` | `false`                      | **Removed** (`true`)    |
+| `DD_TRACE_WORDPRESS_CALLBACKS`            | `false`                      | `true`                  |
+| `DD_TRACE_WORDPRESS_ADDITIONAL_ACTIONS`    | `""`                         | `""`                    |
```

Comment:
What's the change in this line?

Reply:
None. I just kept it to emphasize the fact that it wasn't modified, unlike the two others, but I didn't think about the fact that it can indeed be confusing. Would you prefer it to be removed?

Reply:
Yes, I found it confusing to be honest :-)